### DISCUSSION TITLE
Add SCIM support for user/role/identity management

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#3546] [Ideas]
### Describe the feature or potential improvement

For use/role management specifically, I suggest to use SCIM instead of adding a custom API that does not follow a standard. Let's track interest in this here.

Related to: Admin API (#1007)

### Additional information

https://scim.cloud

**Comment by marcklingen** [comment]
Could evaluate https://www.ory.sh for this

**Comment by flefevre** [comment]
We are also waiting to have SCIM protocol implementing

**Comment by flefevre** [comment]
it seems to be done here : https://langfuse.com/docs/administration/scim-and-org-api
